com.unity.formats.fbx/Editor/FbxPropertyChannelPair.cs

```diff
@@ -216,6 +227,28 @@ private static string GetFbxConstraintSourceProperty(string uniProperty, FbxCons
             return null;
         }
 
+        /// <summary>
+        /// Get the Fbx property name for the given Unity blendshape property name from the given dictionary.
+        /// 
+        /// This is different from GetFbxProperty() because the Unity blendshape properties contain the name
+        /// of the target object.
+        /// </summary>
+        /// <param name="uniProperty"></param>
+        /// <param name="propertyMap"></param>
+        /// <returns>The Fbx property name or null if there was no match in the dictionary</returns>
+        private static string GetFbxBlendshapeProperty(string uniProperty, Dictionary<string, string> propertyMap)
```

Comment:
can you add a flag `applyRegex = false` to GetFbxProperty and put this code in there?

Again here you should be able to use Linq.